awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx

```diff
@@ -1,10 +1,155 @@
-import React, { Component } from 'react';
-import { CardBody } from '@patternfly/react-core';
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import { Project } from '@types';
+import { formatDateString } from '@util/dates';
+import { Button, CardBody, List, ListItem } from '@patternfly/react-core';
+import { DetailList, Detail } from '@components/DetailList';
+import { CredentialChip } from '@components/Chip';
 
-class ProjectDetail extends Component {
-  render() {
-    return <CardBody>Coming soon :)</CardBody>;
+const ActionButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+  & > :not(:first-child) {
+    margin-left: 20px;
   }
+`;
+
+function ProjectDetail({ project, i18n }) {
+  const {
+    allow_override,
+    created,
+    custom_virtualenv,
+    description,
+    id,
+    modified,
+    name,
+    scm_branch,
+    scm_clean,
+    scm_delete_on_update,
+    scm_type,
+    scm_update_on_launch,
+    scm_update_cache_timeout,
+    scm_url,
+    summary_fields,
+  } = project;
+
+  let optionsList = '';
+  if (
+    scm_clean ||
+    scm_delete_on_update ||
+    scm_update_on_launch ||
+    allow_override
+  ) {
+    optionsList = (
+      <List>
+        {scm_clean && <ListItem>{i18n._(t`Clean`)}</ListItem>}
+        {scm_delete_on_update && (
+          <ListItem>{i18n._(t`Delete on Update`)}</ListItem>
+        )}
+        {scm_update_on_launch && (
+          <ListItem>{i18n._(t`Update Revision on Launch`)}</ListItem>
+        )}
+        {allow_override && (
+          <ListItem>{i18n._(t`Allow Branch Override`)}</ListItem>
+        )}
+      </List>
+    );
+  }
+
+  let createdBy = '';
+  if (created) {
+    if (summary_fields.created_by && summary_fields.created_by.username) {
+      createdBy = `${formatDateString(created)} ${i18n._(t`by`)} ${
+        summary_fields.created_by.username
+      }`;
+    } else {
+      createdBy = formatDateString(created);
+    }
+  }
+
+  let modifiedBy = '';
+  if (modified) {
+    if (summary_fields.modified_by && summary_fields.modified_by.username) {
+      modifiedBy = `${formatDateString(modified)} ${i18n._(t`by`)} ${
```

Comment:
This is going to be nit-picky but can we go ahead and wrap the entire thing (as opposed to just `by`) for translation like https://github.com/ansible/awx/pull/4915/files#diff-41634e48aece6599257ec1cf59a0cbb8R111-R114

That lets the translator re-arrange the order of elements depending on the langauage